### PR TITLE
ITM-203:  Ability to have DEV and PROD configurations for Dashboard

### PR DESCRIPTION
**Pull request by brianpippin**
Note:  To run the dashboard locally you will need to use the command in the README with the dev docker-compose file now.

### CODE REVIEW

**Review by kaitlyn-sharo — COMMENTED** [review]
Tested production docker and development docker, and both seemed to work. Not sure if any of my comments are important to change, but Derek and I are both curious about the http vs https questions

File: README.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ docker build --tag dashboard-graphql node-graphql/.
 docker build --tag dashboard-ui dashboard-ui/.
 
 cd docker_setup
+docker-compose -f docker-compose-dev.yml up -d
```
[the remark this answers is not in the document]
Comment (brianpippin) [reply README.md:11]: I actually think we want different scripts so we can use them on production or not.  I created a new folder and moved the startup scrips there along with creating dev ones.

File: dashboard-ui/public/configs/dev/config.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+const API_URL = 'https://localhost:9100/api';
```
[the remark this answers is not in the document]
Comment (brianpippin) [reply dashboard-ui/public/configs/dev/config.js:1]: I had previously set up both to work on the MCS project which is why you are probably seeing both, but as it isn't entirely necessary for us currently I have just updated to http

File: dashboard-ui/public/configs/dev/graphql-config.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const RESET_PASSWORD_URL = "https://localhost:3000/reset-password";
```
[the remark this answers is not in the document]
Comment (brianpippin) [reply dashboard-ui/public/configs/dev/graphql-config.js:1]: I had previously set up both to work on the MCS project which is why you are probably seeing both, but as it isn't entirely necessary for us currently I have just updated to http

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kaitlyn-sharo) [reply README.md:11]: Awesome, thanks. We should probably add those new script locations to the README too

**Review by kaitlyn-sharo — APPROVED** [review]

**Pull request merged**